src/app/groups/groups.component.html

```diff
@@ -30,9 +30,26 @@
       </button>
     </div>
     <div class="col align-self-center">
-      <button mat-raised-button color="primary" (click)="onJoin(group)">
-        Join
-      </button>
+      <div *ngIf="subscribedGroups.includes(group.id); else join">
+        <button
+          mat-raised-button
+          class="btn-custom"
```

Comment:
btn-custom could be changed to w-25 to get almost the same thing. w-25 makes the button a bit wider, but we get rid of the css